README.md

```diff
@@ -30,6 +30,22 @@ docker run --rm -it \
       - .:/src
 ```
 
+### GitLab CI
+
+```yaml
+html proofer:
+  stage: test
+  image:
+    name: klakegg/html-proofer:3.19.2
+    entrypoint: [""]
```

Comment:
Do we need to remove entrypoint just to add `htmlproofer` in the script section?